AccountingSystem/Client/Types/BaseAccountingType.py

```diff
@@ -1,15 +1,14 @@
-# $HeadURL$
+""" Within this module is defined the class from which all other accounting types are defined
+"""
+
 __RCSID__ = "$Id$"
 
-import types
 from DIRAC import S_OK, S_ERROR
 from DIRAC.Core.Utilities import Time
 from DIRAC.Core.DISET.RPCClient import RPCClient
 from DIRAC.AccountingSystem.Client.DataStoreClient import gDataStoreClient
 
-class BaseAccountingType:
-
-  __validDataValues = ( types.IntType, types.LongType, types.FloatType, types.LongType )
+class BaseAccountingType(object):
```

Comment:
class BaseAccountingType( object ):

Reply:
No, this is correct according to pep8: 
https://www.python.org/dev/peps/pep-0008/#id26